setup-tpm2-totp.sh

```diff
@@ -0,0 +1,30 @@
+
+sudo apt -y install build-essential autoconf autoconf-archive automake m4 libtool gcc pkg-config libqrencode-dev pandoc doxygen liboath-dev iproute2 plymouth libplymouth-dev libssl-dev libjson-c-dev libcurl4-openssl-dev
+
+mkdir ./tpm2-totp
+cd tpm2-totp
+
+git clone https://github.com/tpm2-software/tpm2-tss
```

Comment:
Not submodules? Doing it this way means we have to pin a version using `git checkout` or similar, which is doable but this PR doesn't do it.

Alternatively, since you are just `rm -rf`ing at the end, use a release download instead e.g. https://github.com/tpm2-software/tpm2-tss/archive/refs/tags/3.1.0.tar.gz?

Reply:
I think you're right about using submodules. Let's do that.